09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
<p>Add a sentence or two here describing that the MPCD integrator handles <em>both</em> the solvent (via <code>collision_method</code> and <code>streaming_method</code>and solute particles (via <code>methods</code> and <code>forces</code> ).</p><p> </p><p>See a related comment in the first tutorial - this should already be explained in the introduction to the MPCD integrator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>